helper-script.py

```diff
@@ -0,0 +1,247 @@
+import argparse
```

Comment:
I wonder if we can call this script something like `make-prs` or something more meaningful than "helper-script"?